### ISSUE TITLE
process_stats reporting memory incorrectly (condition unclear, perhaps specific container?)

### ISSUE THREAD

**Issue by jdix531** [pixie-io/pixie#456]
**Describe the bug**
Hi, I have an issue where memory info from process_stats table (rss_bytes / vsize_bytes) seems incorrect for at least one container in a manner which seems not particularly predictable or clearly reproducible (happens occasionally over time). For example, I am reported that the rss_bytes of a container (parc_logger_api, pictured below in Figure 1 and unnamed but the 1.2TB in figure 2) total 1.2 TB where this is impossible on nodes with 32 GB max memory. Additionally, that value is much larger than the vsize_bytes which shouldn't be possible. Worth noting, vsize_bytes doesn't seem accurate at all times either but the reported values are much more sane (at most a couple hundred MB off).

Some correlations:
- Exporting these metrics seems to crash the query exporter in a way that data no longer is sent out through pixie cloud or the open telemetry plugin scripts (pixie cloud reports server error: stream timeout). Deleting the query exporter pod seems to allow it to pick back up as normal. This seems to not be happening 100% but after some time while preforming memory-exporting scripts.
- the cluster reports healthy but doesn't return queries during this time. I captured logs after this started occurring (by about 45 min, last crash ~1:53p est)
- vsize fluctuations into incorrect values seem to occur at the same time as the rss is also incorrect.
- vsize and rss do not seem to change at all in the pixie ui but perhaps data that goes to open telemetry plugin is correct only sometimes. (Figure 2)

I'd like to outline what I've seen, what I've tried, and hopeful get a little help on reproduction.

**To Reproduce**

**Expected behavior**
Memory reported in pixie ui should be accurate for containers.

**Screenshots**

Figure 1
note: rss on left, vsize on right.
![Screen Shot 2022-06-08 at 10 29 50 AM](https://user-images.githubusercontent.com/5290156/172661107-eaeed8ba-17cf-479f-84be-69c4e13e4b46.png)
Figure 2
![Screen Shot 2022-06-02 at 1 52 18 PM](https://user-images.githubusercontent.com/5290156/172662501-414d2fd7-f12a-4e41-8742-76a497d2dd5e.png)

**Logs**
Please attach the logs by running the following command:
```
./px collect-logs
```
[pixie_logs_20220608144618.zip](https://github.com/pixie-io/pixie/files/8864273/pixie_logs_20220608144618.zip)

**App information (please complete the following information):**
- Pixie version - 0.11.2
- K8s cluster version - 1.19.15
- Node Kernel version - 5.4.188-104.359.amzn2.x86_64
- Browser version - Firefox 101.0

- px cli version - 0.7.11+Distribution.0965870.20220607181722.1

**Additional context**

**Comment by yzhao1012** [comment]
Thanks for reporting. No clue of what's going on. 

* Could you copy the `/proc` output with `cat /proc/<PID>/status` and copy the output to this thread?
* Create a `mem.pxl` file with the following content:
```
import px
df = px.DataFrame(table='process_stats', start_time='-10s')
df.ns = df.ctx['namespace']
df.pod = df.ctx['pod']
df = df[px.contains(df.pod, 'px-operator')]
px.display(df)
```
And run `px run -f mem.pxl`.

**Comment by chengruizhe** [comment]
This issue has been fix in 4ddc45d. The fix was to correctly account for the spaces in the command name field in `proc/pid/stat`, such as `npm start`.
